spec/active_record/annotate/file_spec.rb

```diff
@@ -52,6 +52,18 @@ class User < ActiveRecord::Base
 #   t.integer :age, null: false, default: 0
 # end
 
+class User < ActiveRecord::Base
+  has_many :posts
+end
+    FILE
+  end
+  
+  let(:expected_ignored_result) do
+    <<-FILE
+# enCoding: utf-8
+# frozen_string_literal: true
+# warn_indent: true
+
 class User < ActiveRecord::Base
   has_many :posts
 end
```

Comment:
This is not the expected result here as the ignored file will not be touched at all - the gem won't erase the old annotation. [This](https://github.com/7even/active_record-annotate/blob/86a26ed5de2f534b7278004ba709db82749bb666/spec/active_record/annotate/file_spec.rb#L8-L19) is what should be expected.

Reply:
But since this is a wrong place to test `ignored_models` you can just delete this.